front/src/pages/student-registration-page/index.js

```diff
@@ -10,71 +13,29 @@ const layout = {
   wrapperCol: { span: 16 },
 };
 
-const tailLayout = {
-  wrapperCol: {
-    offset: 9,
-    span: 16,
-  },
-};
-
 class StudentRegistrationPage extends Component {
   constructor(props) {
     super(props);
     this.handleSubmit = this.handleSubmit.bind(this);
-    this.validateEng = this.validateEng.bind(this);
-    this.validateUkr = this.validateUkr.bind(this);
-    this.validatePhoneNumber = this.validatePhoneNumber.bind(this);
-    this.confirmPassword = this.confirmPassword.bind(this);
+    this.renderStudentRegistrationForm = this.renderStudentRegistrationForm.bind(this);
   }
 
-  handleSubmit(data) {
-    data.groupToken = this.props.token;
+  componentDidUpdate() {
```

Comment:
I think it should be `componentDidMount`

Reply:
I don't know. If I will use componentDidMount it will work only after component will appear. I need this notification every time after I am getting response from server.

Reply:
I will remove notification logic to students.saga.js file